### PR TITLE
Mockbee

### PR DESCRIPTION
**Pull request by DivyanshSareen**
Implemented mockbee backend. Added custom data for categories and products.

files other than mockbee code https://github.com/DivyanshSareen/plant-shop/pull/6/commits/ba676aeaabc20f19c9b27c313e1edce537ad3c01

### CODE REVIEW

**Review by sanjitsarkar — APPROVED** [review]
Great work could not find any issues.
File structure is also good. Code is readable and clean.
Great work.

File: src/backend/db/products.js [hunk 1]
```diff
@@ -8,23 +8,173 @@ import { v4 as uuid } from "uuid";
...
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    name: "balsam",
+    categoryName: "perennial",
+    image: "/perennial/balsam.webp",
+    price: "100",
+    discount: "10",
```
Comment (Shubhamkr2610) [inline src/backend/db/products.js:15]: suggestion: price shouldn't be in double quote because you have to do mathematical operation on it

**Review by Shubhamkr2610 — COMMENTED** [review]
everything looks good . you can merge it

File: src/backend/db/products.js (hunk 1 above)
Comment (DivyanshSareen) [reply src/backend/db/products.js:15]: Thanks for the suggestion, I will surely fix this

**Pull request merged**